### DISCUSSION TITLE
<喜歡的話請響鈴>

### DISCUSSION THREAD

**Discussion by Sylvie0601** [1101-GS4719-C/Python-Community-GS4719-C#1094] [General]
2016年推出的韓國漫改劇<喜歡的話請響鈴>，劇情圍繞著一個名叫「戀愛鈴」的應用程式，在這個世界幾乎所有人都有這個應用程式，他可以感知到周圍人對自己是否有好感，當暗戀自己的人進入半徑10公尺之內，戀愛鈴就會發出提醒，若是互相喜歡的兩人都有開啟應用程式，就能夠敲響對方的戀愛鈴。
拋開他的戀愛劇情的話，如果戀愛鈴被研究出來的話我會覺得很酷ㄟ，但是他感覺就會失去戀愛前人與人之間曖昧的這種張力，跟若是發現被一個自己討厭的人喜歡等等的問題，這個漫畫跟電視劇都有寫道，個人很喜歡第一季，可以在Netflix看!
歡迎下方大家討論!
附上預告連結:https://www.youtube.com/watch?v=9sKbVzc8yic

**Comment by Jasper525** [comment]
這讓我想到台灣社交距離的軟體，觀念其實蠻類似的，所以以現在的技術確實可能開發出來。交友軟體已經有引入用手機gps定位找配對的類似功能了。

**Comment by yuxinlin93** [comment]
這樣蠻方便的，但少了那個曖昧的階段好像真的蠻可惜的。不過那一些追求效率、想要快速配對的男女可能會很喜歡這種技術
軟體，不需要花時間試探揣測，你喜歡我我喜歡你，那就不要廢話直接在一起了吧，這種效率感

**Comment by Guagua1234** [comment]
我認為在曖昧之中捉摸不定的感覺最有談戀愛的味道，因為一個人而心情起伏不定，如果少了這種感覺就不叫談戀愛了阿，但也會有人走效率派，最讓我好奇的是程式要怎麼知道自己喜歡誰呢，如果是自己設定的話，感覺就有點失去真實性了。

**Comment by Valentinaa0214** [comment]
我有看過這部！！那時候在追劇時就覺得超酷，其實蠻類似現代交友軟體的形式，只是更加客觀的忠於自己的內心(？
我覺得這個軟體的發想理念挺好的，適合超有效率的開啟戀愛方式

**Comment by wenchi9023** [comment]
好有趣的劇情! 
雖然能更快速讓彼此知道是否互有好感，但這樣就會少了臉紅心跳的互相試探的曖昧階段
如果這項技術真的被研發出來，我想我也不會用，如果是一個我討厭的人喜歡我，我應該會更不知道該如何和這個人互動 :（
